src/jit/emitxarch.cpp

```diff
@@ -3798,6 +3800,82 @@ void emitter::emitIns_R_R_I(instruction ins, emitAttr attr, regNumber reg1, regN
     emitCurIGsize += sz;
 }
 
+void emitter::emitIns_R_A(
```

Comment:
@CarolEidt, Just as a note. We already have several similar `emitIns_R_A` methods, but they are `emitIns_R_AI`, `emitIns_R_AX`, etc. They only differ based on which of `baseReg, indxReg, scale, and offs` are passed in.

I found it significantly easier to have a single method which takes and assigns all the args. This works because when we extract these from an Indir node, the unused ones are set to the correct value (reg = REG_NA, scale = 1, offs = 0)

Reply:
Would it make sense to have the others call this, and/or phase them out?

Reply:
Yes, I believe so. I'll log a bug.

Reply:
https://github.com/dotnet/coreclr/issues/15828